include/clap/ext/draft/background-operation.h

```diff
@@ -19,6 +19,8 @@ improve the loading time.
 Concurrency isn't a trivial thing, and this extension should only be implemented if the plugin has
 expensive tasks to perform during load state or activate.
 
+Implementing this extension implies that plugin->activate() can be called from a [bgop-thread].
```

Comment:
I think maybe you could make this comment a bit more obvious. At the moment it's easy to miss, for quite an important behavioural detail.